src/GeositeFramework/region.json

```diff
@@ -11,20 +11,22 @@
     },
     "headerLinks": [
         { "text": "Azavea", "url": "http://www.azavea.com/" },
-        { "text": "GIS", "url": "http://en.wikipedia.org/wiki/Geographic_information_system" }
-    ],
-    "regionLinks": [
```

Comment:
Is this key no longer supported in the config?  If so, we'll want to update the JSON schema and also the wiki documentation for the region.json file - noting that there is a special `items` key that can nest additional links.